locales/de.json

```diff
@@ -2086,7 +2086,9 @@
 			"importing": "importiere",
 			"importSubmission": "Abgabe des Schülers/ der Schülerin importieren",
 			"onTheSubject": "Zum Thema",
+			"saveDraft": "Entwurf anlegen",
```

Comment:
Why is it "create" in german and "save" in english and other languages?

(As a User "Speichern" would tell me more about what is happening than "Anlegen" which is kind of vague for me.)

Reply:
@bergatco guter Punkt habe ich überlesen.

Reply:
I agree, that this wording is misleading. You can click the button several times and it always updates the draft, right? But the wording sounds like you would always a create a new one.

Reply:
Fixed by renaming "Entwurf anlegen" to "Entwurf speichern"!